homeworks/05-redux-basic/package.json

```diff
@@ -0,0 +1,53 @@
+{
+  "name": "05-redux-basic",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@testing-library/jest-dom": "^5.11.4",
```

Comment:
Not sure why Create React App haven't fixed this yet in their code but testing libraries and types should be in `devDependencies`.